### ISSUE TITLE
Create a notebook as baseline knowledge to document GALEX dataset data access in cloud using existing tools/services/package

### ISSUE THREAD

**Issue by sshen-stsci** [fornax-navo/fornax-s3-subsets#3]
We need to provide baseline knowledge for Galex dataset S3 subsetting . The notebook should specify the following items:

- Specify the nature of the Galex dataset and GPhoton data
- The general approach for handling Galex dataset S3 subsetting
- The data location in cloud and how to access them
- Possible use cases for Galex dataset

**Comment by jaymedina** [comment]
Hi @sshen-stsci , is this notebook meant to explain how to access GALEX cutouts, or just GALEX data in general? Currently the `astroquery.mast` cloud feature allows you to pull GALEX datasets from the cloud, but I don't think those are cutouts, and I don't think there's current functionality on Astrocut to take cutouts from GALEX.

**Comment by sshen-stsci** [comment]
@jaymedina You can document where is the GALEX data in cloud and how we access it through the astroquery package or other tools/service. The GALEX cutout is not available for us. The notebook mainly will document what we have and we can do now as baseline knowledge for future S3 subsetting development. Hope it makes sense to you.

**Comment by jaymedina** [comment]
Yes, that makes sense. I will be available to tackle this in the next sprint. Thanks!
